Podfile

```diff
@@ -0,0 +1,24 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'exam001' do
+  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for exam001
+  pod 'Firebase/Core'
+  pod 'Firebase/Database'
+  pod 'Firebase/Auth'
+  pod 'Alamofire', '~> 5.0.0.beta.1'
+
+  target 'exam001Tests' do
+    inherit! :search_paths
+    # Pods for testing
+  end
+
+  target 'exam001UITests' do
+    inherit! :search_paths
+    # Pods for testing
+  end
+
+end
```

Comment:
Please do some research to understand the following:
What does target means?
What this block `~> 5.0.0.beta.1'` in `'Alamofire', '~> 5.0.0.beta.1'` represents?
How to point to pod branches?
How to pull specific branch into our project and update?
How to point to a local repo, and point to a specific local branch?